spec/fixtures/cookbooks/image_streamer_test_api1200.rb/attributes/default.rb

```diff
@@ -0,0 +1,19 @@
+#
```

Comment:
Name of the folder is incorrect here. remove ".rb from the folder name: image_streamer_test_api1200.rb"

Reply:
done